glean-core/src/lib.rs

```diff
@@ -611,6 +614,11 @@ impl Glean {
         Ok(())
     }
 
+    /// Set internally-handled application lifetime metrics.
+    fn set_application_lifetime_metrics(&self) {
```

Comment:
```suggestion
    fn set_application_lifetime_core_metrics(&self) {
```
To make it clear that this is about setting core stuff.